src/data/reports/glass-data-submission/GLASSDataSubmissionDefaultRepository.ts

```diff
@@ -84,9 +92,15 @@ export class GLASSDataSubmissionDefaultRepository implements GLASSDataSubmission
         const objects =
             (await this.globalStorageClient.getObject<GLASSDataSubmissionItem[]>(namespace))?.filter(object => {
                 const amrModule = modules.find(module => module.name === "AMR")?.id;
+                const amrIndividualModule = modules.find(module => module.name === "AMR - Individual")?.id;
+                const egaspModule = modules.find(module => module.name === "EGASP")?.id;
 
-                return object.module === amrModule;
+                if (module === "AMR") return object.module === amrModule;
```

Comment:
This ifs may be equivalent to `(module === "AMR" && object.module === amrModule) || (module === "AMRIndividual" && object.module === amrIndividualModule) || (module === "EGASP" && object.module === egaspModule)` (if you want them to be filtered those `object` that fulfill this and if not, do not filter them)